### ISSUE TITLE
Msys2 Install OK but can't install Hamlib

### ISSUE THREAD

**Issue by mm5agm** [KI7MT/jtsdk-dotnet-core#1]
Installed Mysys2 as per instructions and got the following after running $ pacman -Syuu a few times
********************************
$ pacman -Syuu
:: Synchronising package databases...
 mingw32 is up to date
 mingw64 is up to date
 msys is up to date
:: Starting core system upgrade...
 there is nothing to do
:: Starting full system upgrade...
 there is nothing to do
******************************
Used exit to leave Msys2 and re-opened it
Put in the command
pacman -S --needed --noconfirm - < /usr/etc/hamlib-pkg-list.txt
and got
**********************************
Colin@Colin-PC MSYS ~
$ pacman -S --needed --noconfirm - < /usr/etc/hamlib-pkg-list.txt
error: target not found: apr
error: target not found: apr-util
error: target not found: autoconf
error: target not found: automake-wrapper
error: target not found: doxygen
error: target not found: gettext-devel
error: target not found: git
error: target not found: libtool
error: target not found: swig
error: target not found: libreadline-devel
error: target not found: libxml2-devel
error: target not found: make
error: target not found: libgdbm-devel
error: target not found: pkg-config
error: target not found: texinfo

Colin@Colin-PC MSYS ~
******************************

**Comment by KI7MT** [comment]
Hi Colin,

This may be a bit slower, as it install each package individually, but try the following and see if the packages install:

``` shell
while read p; do pacman -S --needed --noconfirm "$p"; done < /usr/etc/hamlib-pkg-list.txt
```
If it works, I'll most likely add a script to perform this action for users.

Thanks
Greg

**Comment by mm5agm** [comment]
Hi Greg,
I get exactly the same result  running the while read.
Colin

**Comment by nzampella** [comment]
Yep .. me too .. went the individual install route.   Evidently the command doesn't like the input for some reason.

Neil, KN3ILZ

**Comment by KI7MT** [comment]
Hi Colin, Neil, 

This is a strange problem. I wrote the source file inside an MSYS2 shell with nano, and both methods work for me on a new MSYS2 install.

I'll leave this open until I write the shell script, and update the [MSYS2 Wiki Page] (https://github.com/KI7MT/jtsdk-dotnet-core/wiki/Install-Msys2) to reflect the new method of install.

Thanks
Greg

**Comment by nzampella** [comment]
Greg,

did you really mean 'user should NOT' or 'user should' ??


Neil

**Comment by KI7MT** [comment]
Hi Neil,

I caught that typo on closing the ticket. I corrected it, but, it seems to have not gone out. It should say

"user should now issue the following command: install-pkg-list.sh"

In other words, just run the script :-)

73's
Greg, KI7MT
